### PR TITLE
Fix `full-cover-fit` setting in the editor

### PR DESCRIPTION
**Pull request by iainsmacleod**
Should cover-fit in line 18 not be full-cover-fit? e.g., const OptionsArtwork = ['cover', 'full-cover', 'material', 'full-cover-fit', 'none'];

### CODE REVIEW

**Comment by kalkih** [comment]
Sorry for the long wait.
Yes, you are correct!

Can you rebase this on the dev branch?

**Comment by Drafteed** [comment]
Please rebase from the latest dev branch. I'm closing this. Open it again if necessary.

**Comment by iainsmacleod** [comment]
ok...so I am really new to doing this, so let me know if I did something wrong. I cloned my repo locally, made the changes, then did a rebase to the dev branch... so I assume you are good to pull these into the master.

again, feel free to correct/educate me if I did something wrong

File: src/editor.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const fireEvent = (node, type, detail = {}, options = {}) => {
   return event;
 };
 
-const OptionsArtwork = ['cover', 'full-cover', 'material', 'cover-fit', 'none'];
+const OptionsArtwork = ['cover', 'full-cover', 'material', 'full-cover-fit', 'none'];
```
Comment (Drafteed) [inline src/editor.js:18]: Let's sort it a little bit:

```suggestion
const OptionsArtwork = ['cover', 'full-cover', 'full-cover-fit', 'material', 'none'];
```

**Review by Drafteed — APPROVED** [review]
Thanks, @iainsmacleod 👍

**Comment by iainsmacleod** [comment]
Happy to assist, @Drafteed - and thank you.

**Pull request merged**